File: 프로그래머스/unrated/181908. 접미사인지 확인하기/접미사인지 확인하기.py
def solution(my_string, is_suffix):
    # 슬라이싱 사용
    # my_string의 맨 끝으로부터 is_suffix의 글자 수만큼의 문자열이 is_suffix와 같은지 비교
    return 1 if my_string[-len(is_suffix):] == is_suffix else 0

File: 프로그래머스/unrated/181908. 접미사인지 확인하기/접미사인지 확인하기.js
function solution(my_string, is_suffix) {
    // my_string의 맨 끝으로부터 is_suffix의 글자 수만큼의 문자열이 is_suffix와 같은지 비교
    return my_string.slice(-is_suffix.length) === is_suffix ? 1 : 0
}